test/CMakeLists.txt

```diff
@@ -891,6 +893,18 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_sondetac
 # MISC converters
 #===============================================================================
 
+ecbuild_add_test( TARGET  test_iodaconv_amsr2_gcom-w1
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND bash
+                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                          netcdf
+                  "${CMAKE_BINARY_DIR}/bin/amsr2_2ioda.py
+                  -i testinput/amsr2_gcom-w1_20220216.h5
+                  -o testrun/amsr2_gcom-w1_20220216.nc4
+                  -d 2022021600"
+                  amsr2_gcom-w1_20220216.nc4 ${IODA_CONV_COMP_TOL_ZERO} )
```

Comment:
@smaticka judging from the ctests this probably too tight of a tolerance.

Reply:
What does tolerance mean? the size of the files?

Reply:
all tests passed for me

Reply:
It only happened on intel compilers. Looks like a floating point thing. @fcvdb will talk to you about what's needed.